recipe/meta.yaml

```diff
@@ -66,6 +66,8 @@ about:
 extra:
   recipe-maintainers:
     - conda-forge/r
+    - mxr-conda
```

Comment:
this is not activating the windows build... you have to remove the skip in line 17.

Reply:
I received a request from a maintainer on r-gensa about not just listing that I'm adding new maintainers as the PR message but to be more descriptive with my intentions with the PR. 

Hence the name for the PR. Please see [message](https://github.com/conda-forge/r-gensa-feedstock/pull/8#issuecomment-1009027636)

Reply:
@dbast The cleaned recipe for r-eulerr that was generated by the r-skeleton-helper was added to staged recipes and the skip line in the meta.yaml for r-eulerr feedstock has been removed.